cetmix_tower_aws/__manifest__.py

```
@@ -0,0 +1,24 @@
+# Copyright 2024 Cetmix OÜ
```

Pylint (4.0.4)

[error] Unrecognized option found: readme-template-url, manifest-required-authors, manifest-required-keys, manifest-deprecated-keys, license-allowed, valid-odoo-versions

(E0015)